worlds/tunic/rules.py

```diff
@@ -312,7 +312,9 @@ def set_location_rules(world: "TunicWorld", ability_unlocks: Dict[str, int]) ->
 
     # Swamp
     set_rule(multiworld.get_location("Cathedral Gauntlet - Gauntlet Reward", player),
-             lambda state: state.has(laurels, player) and state.has(fire_wand, player) and has_sword(state, player))
+             lambda state: (state.has(fire_wand, player) and has_sword(state, player))
```

Comment:
I don't believe these parens are necessary
```suggestion
             lambda state: state.has(fire_wand, player) and has_sword(state, player)
```

Reply:
They're not, I just feel weird not putting them in -- makes the intent much clearer imo